examples/lightgbm/distributed-training.ipynb

```diff
@@ -81,7 +81,11 @@
    },
    "outputs": [],
    "source": [
-    "lightgbm.execute(config=params, docker_registry=DOCKER_REGISTRY)"
+    "lightgbm.execute(config=params,\n",
+    "                          docker_registry=DOCKER_REGISTRY,\n",
+    "                          cores_per_worker=1,\n",
```

Comment:
Does cores_per_worker specify #cores assigned to each worker or to allocate a single core node?

Reply:
#cores assigned to each worker . Added doc strings.